src/Paket.Core/PaketConfigFiles/DependenciesFile.fs

```diff
@@ -262,17 +264,79 @@ type DependenciesFile(fileName,groups:Map<GroupName,DependenciesGroup>, textRepr
+                        |> Seq.map snd
+                        |> Seq.map (fun p ->
+                            let oldDepsInfo = step1Deps |> Seq.tryFind (fun d -> d.Name = p.Name)
+                            { Name = p.Name
+                              VersionRequirement = VersionRequirement (VersionRange.Exactly p.Version.AsString, PreReleaseStatus.All)
+                              // How to get that for the package?
+                              ResolverStrategyForDirectDependencies =
+                                match oldDepsInfo with
+                                | Some d -> d.ResolverStrategyForDirectDependencies
+                                | None -> group.Options.ResolverStrategyForDirectDependencies
+                              ResolverStrategyForTransitives =
+                                match oldDepsInfo with
+                                | Some d -> d.ResolverStrategyForTransitives
+                                | None -> group.Options.ResolverStrategyForTransitives
+                              Parent = PackageRequirementSource.DependenciesFile "runtimeresolution.dependencies"
+                              Graph = []
+                              Sources = group.Sources
+                              Settings =
+                                match oldDepsInfo with
+                                | Some d -> d.Settings
+                                | None -> group.Options.Settings })
+                        |> fun des -> Seq.append des runtimeDeps
+                        |> Seq.distinctBy (fun p -> p.Name) |> Set.ofSeq
+                        // works, alternatively
+                        //(step1Deps |> Set.toList) @ runtimeDeps |> Seq.distinctBy (fun p -> p.Name) |> Set.ofSeq
+
+                    if Environment.GetEnvironmentVariable "PAKET_DEBUG_RUNTIME_DEPS" = "true" then
+                        tracefn "Runtime dependencies: "
+                        let runtimeDepsFile = DependenciesFile.FromSource ""
+                        let runtimeDepsFile = runtimeDepsFile.AddFrameworkRestriction(Constants.MainDependencyGroup, getRestrictionList group.Options.Settings.FrameworkRestrictions)
+                        let runtimeDepsFile = runtimeDepsFile.AddResolverStrategyForTransitives(Constants.MainDependencyGroup, group.Options.ResolverStrategyForTransitives)
+                        let runtimeDepsFile = (runtimeDepsFile:DependenciesFile).AddResolverStrategyForDirectDependencies(Constants.MainDependencyGroup, group.Options.ResolverStrategyForDirectDependencies)
+                        let runtimeDepsFile =
+                            runtimeResolutionDeps
+                            |> Seq.fold (fun (deps:DependenciesFile) dep -> deps.AddAdditionalPackage(Constants.MainDependencyGroup, dep.Name, dep.VersionRequirement, dep.ResolverStrategyForTransitives, dep.Settings)) runtimeDepsFile
+
+                        tracefn "Depsfile: \n%O" runtimeDepsFile
```

Comment:
This helps printing out the deps file we resolve when handling runtime dependencies, this might help in the future...